src/definitions.ts

```diff
@@ -95,6 +95,23 @@ export interface CoreSettings {
   urls?: CloudSettingsUrls;
 }
 
+export interface AuthOptions {
```

Comment:
The comments for these are in the wrong spot and the documentation won't be generated correctly